### PR TITLE
Add documentation

### LINKED ISSUE #95: Add documentation
- [ ]  Create a readme with `.env.example`
- [ ]  Describe the steps for running this project

### PR DESCRIPTION
**Pull request by paulotijero**
Resolve #95

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+|  ADMIN  |     davis.con.fab@gmail.com      | secret123 |
+|  ADMIN  |        liamrn94@gmail.com        | secret123 |
+|  ADMIN  |     mperezzevallos@gmail.com     | admin123  |
+|  ADMIN  |      paulotijero@gmail.com       | admin123  |
+| REGULAR | davis.con.fab+regular@gmail.com  | secret123 |
+| REGULAR |    liamrn94+regular@gmail.com    | secret123 |
+| REGULAR | mperezzevallos+regular@gmail.com |  regular  |
+| REGULAR |  paulotijero+regular@gmail.com   |  regular  |
+
+## This is a routes API's
```
Comment (liannivin) [inline documentation.md]: Hey, @paulotijero. What's going on with the routes?

File: documentation.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Documentation
+
+## How do install
```
Comment (valevassallo) [inline documentation.md]: "How to install:"

File: documentation.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+You need install the Gemfile
```
Comment (valevassallo) [inline documentation.md]: "You need to install the Gemfile"

File: documentation.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+bundle install
+```
+
+Now run the migrates and seeds
```
Comment (valevassallo) [inline documentation.md]: the migrations*

File: documentation.md (hunk 1 above)
Comment (paulotijero) [reply documentation.md]: I'm adding them :sweat_smile:

File: documentation.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+rails db:setup
+```
+
+Don't forget add `.env`, you can copy a template of [.env.example](https://github.com/codeableorg/rails-booking-clon/blob/master/.env.example)
```
Comment (valevassallo) [inline documentation.md]: "Don't forget to add the .env file" "copy ... from"

File: documentation.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## This is the list user's
```
Comment (valevassallo) [inline documentation.md]: "This is the list of users:"

File: documentation.md (hunk 1 above)
Comment (valevassallo) [inline documentation.md]: "These are the routes for the API"

File: documentation.md (hunk 2 above)
Comment (paulotijero) [reply documentation.md]: Thank a lot, Val!

**Comment by paulotijero** [comment]
Thank for review my English, @valevassallo

File: documentation.md (hunk 3 above)
Comment (valevassallo) [reply documentation.md]: or better yet,,, "You need to install the gems from the Gemfile"

File: documentation.md (hunk 1 above)
Comment (MariethPerez) [reply documentation.md]: Good job !!! :)

**Pull request merged**